src/sentry/api/serializers/models/grouptombstone.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import
+
+from itertools import izip
+import six
+
+from sentry.api.serializers import Serializer, register, serialize
+from sentry.constants import LOG_LEVELS
+from sentry.models import (
+    GroupTombstone, User
+)
+
+
+@register(GroupTombstone)
+class GroupTombstoneSerializer(Serializer):
+
+    def get_attrs(self, item_list, user):
+        project_set = set(i.project for i in item_list)
+        projects = {
+            p.id: d for p, d in izip(project_set, serialize(project_set, user))
```

Comment:
Ah that makes sense. Actually, would this not always be the same project for all `GroupTombstone`s passed in, or do we want to account for potentially using this serializer organization-wide?